### PR TITLE
Release/config changes

### PR DESCRIPTION
**Pull request by carlbm**
Mikes changes to add an app that can help with setting up the SF config
Dave S changes to help setup dev machine

### CODE REVIEW

**Comment by carlbm** [comment]
I'm happy with the changes, but it's my PR so can't approve it...

File: src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 1]
```diff
@@ -13,8 +13,8 @@
     <Parameter Name="LogLevel" DefaultValue="Debug" />
     <Parameter Name="ConsoleLogLevel" DefaultValue="" />
     <Parameter Name="ProcessMessageSequentially" DefaultValue="false" />
-    <Parameter Name="RequiredPaymentsService_EndpointName" DefaultValue="" />
-    <Parameter Name="RequiredPaymentsService_FailedMessagesQueue" DefaultValue="" />
+    <Parameter Name="RequiredPaymentsService_EndpointName" DefaultValue="sfa-das-payments-requiredpayments" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 2]
```diff
@@ -16,8 +16,8 @@
     <Parameter Name="LogLevel" DefaultValue="Information" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 3]
```diff
@@ -16,8 +16,8 @@
...
     <Parameter Name="ConsoleLogLevel" DefaultValue="" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.PeriodEnd.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 4]
```diff
@@ -2,18 +2,18 @@
 <ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="SFA.DAS.Payments.PeriodEnd.ServiceFabricType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
     <Parameter Name="PeriodEndService_InstanceCount" DefaultValue="-1" />
-    <Parameter Name="PeriodEndService_EndpointName" DefaultValue="" />
-    <Parameter Name="PeriodEndService_FailedMessagesQueue" DefaultValue="" />
-    <Parameter Name="SubscriptionName" DefaultValue="" />
-    <Parameter Name="JobStatusQueueName" DefaultValue="" />
-    <Parameter Name="AuditQueueName" DefaultValue="" />
+    <Parameter Name="PeriodEndService_EndpointName" DefaultValue="sfa-das-payments-periodend" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.PeriodEnd.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 5]
```diff
@@ -13,8 +13,8 @@
     <Parameter Name="LoggingConnectionString" DefaultValue="" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 6]
```diff
@@ -14,11 +14,11 @@
     <Parameter Name="LogLevel" DefaultValue="Information" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.EarningEvents.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 7]
```diff
@@ -2,11 +2,11 @@
 <ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="SFA.DAS.Payments.EarningEvents.ServiceFabricType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
     <Parameter Name="EarningEventsService_InstanceCount" DefaultValue="-1" />
-    <Parameter Name="EarningEventsService_EndpointName" DefaultValue="" />
-    <Parameter Name="EarningEventsService_FailedMessagesQueue" DefaultValue="" />
-    <Parameter Name="SubscriptionName" DefaultValue="" />
-    <Parameter Name="JobStatusQueueName" DefaultValue="" />
-    <Parameter Name="AuditQueueName" DefaultValue="" />
+    <Parameter Name="EarningEventsService_EndpointName" DefaultValue="sfa-das-payments-earningevents" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.EarningEvents.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

File: src/SFA.DAS.Payments.Datalocks.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 8]
```diff
@@ -3,10 +3,10 @@
   <Parameters>
     <Parameter Name="ApprovalsService_InstanceCount" DefaultValue="-1" />
     <Parameter Name="DataLockStatusChangedService_InstanceCount" DefaultValue="-1" />
-    <Parameter Name="DataLockStatusChangedService_EndpointName" DefaultValue="" />
-    <Parameter Name="DataLockStatusChangedService_FailedMessagesQueue" DefaultValue="" />
-    <Parameter Name="DataLockEventService_EndpointName" DefaultValue="" />
-    <Parameter Name="DataLockEventService_FailedMessagesQueue" DefaultValue="" />
+    <Parameter Name="DataLockStatusChangedService_EndpointName" DefaultValue="sfa-das-payments-datalock-status-change" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.Datalocks.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: If we are setting defaults in this file we should set default db connection strings, console log, etc.

**Comment by ESFA-Pawan** [comment]
Merged as requested from Carl - There are no production changes, it's all dev config
It's just dev config helpers, no production code so no issues with merges.

**Pull request merged**